File: src/Services/StudentService/StudentService.Domain/Student.cs

using Education.Core.Domain;
using StudentService.Domain.Enums;

namespace StudentService.Domain;

public class Student : BaseEntity
{
    public StudentStatus Status { get; set; } = StudentStatus.NoData;
    public PersonalInformation PersonalInformation { get; set; }
    public InformationBySchool InformationBySchool { get; set; }
    public List<StudentEducationProgram> EducationPrograms { get; set; }
}

File: src/Webs/sinhvien/src/domain/student.ts
import { StudentEducationProgram } from "@/domain/student_education_program.ts";
import {PersonalInformation,  } from "./personal_information";
import { InformationBySchool } from "./information_by_school";

export type Student = {
  personalInformation: PersonalInformation;
  informationBySchool: InformationBySchool;
  educationPrograms: StudentEducationProgram[];
  status: number;
  id: string;
  createdAt: string;
  updatedAt: string | null;

}
